Software/src/qtsingleapplication/common.pri

```diff
@@ -1,4 +1,4 @@
-exists(config.pri):infile(config.pri, SOLUTIONS_LIBRARY, yes): CONFIG += qtsingleapplication-uselib
+#exists(config.pri):infile(config.pri, SOLUTIONS_LIBRARY, yes): CONFIG += qtsingleapplication-uselib

```

Comment:
When running `qmake`, this throws an error that it cannot find `config.pri`. Which is weird, since it's checked before trying to include it by using `exists`.

The reason why I commented it out, is because I moved my personal builds to AppVeyor, and [AppVeyor apparently doesn't like output in stderr](https://ci.appveyor.com/project/Archomeda/lightpack/build/5.11.2.1-1/job/7gdthqrlvxwalg3f). So this line prevented the build from continuing. And because this line was originally from the qtsingleapplication source, I decided to not remove it. I will take a look if I can somehow make it working the way it was supposed to.